# HG changeset patch
# User Andrew Cooper <andrew.cooper3@citrix.com>
# Date 1377595743 -7200
# Node ID 10c2d9849207c7fe89e1a0e3f28b66b50aae76ae
# Parent  93d4ee2987b5fe561d92da6bc6a8edbbf86a03e0
fix gdbstub build c/s c8177e691f

That changeset moved the watchdog functions from nmi.h to their own
watchdog.h.  I thought I had updated all relevant header files and the
compiler was happy as well.  However, gdbstub is not even compiled by default,
and I accidentally missed it.

Signed-off-by: Andrew Cooper <andrew.cooper3@citrix.com>

diff -r 93d4ee2987b5 -r 10c2d9849207 xen/common/gdbstub.c
--- a/xen/common/gdbstub.c
+++ b/xen/common/gdbstub.c
@@ -38,7 +38,7 @@
 #include <xen/spinlock.h>
 #include <xen/serial.h>
 #include <xen/irq.h>
-#include <xen/nmi.h>
+#include <xen/watchdog.h>
 #include <asm/debugger.h>
 #include <xen/init.h>
 #include <xen/smp.h>
